packages/rum-core/src/domain/view/viewMetrics/trackLargestContentfulPaint.ts

```diff
@@ -58,6 +58,7 @@ export function trackLargestContentfulPaint(
     { capture: true, once: true }
   )
 
+  let prevLcpSize = 0
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** to be more explicit and consistent with the comment‏
```suggestion
  let biggestLcpSize = 0
```